_sass/_base.scss

```diff
@@ -124,27 +137,6 @@ img {
   max-width: 100%;
 }
 
-blockquote {
```

Comment:
Why delete the styling for a blockquote?

Reply:
I wrote a new one (see template post to see how it renders blockquotes):

```css
  blockquote {
      background-color: #eee;
      padding: 0px 52px;
      margin: 8px 0;
      border-radius: 4px;
      position: relative;
      overflow: hidden;
      font-size: 18px;

      &::before,
      &::after {
          content: '“';
          color: #000;
          font-size: 120px;
          position: absolute;
          line-height: 1;
          opacity: 0.2;
      }
      &::before {
          top: -16px;
          left: -4px;
      }

      &::after {
          bottom: -16px;
          right: -4px;
          transform-origin: center;
          transform: rotate(180deg);
      }
  }
```